### DISCUSSION TITLE
能否实现 Obsidian 退出 vim_mode 后自动切换回中文

### DISCUSSION THREAD

**Discussion by yaoyhu** [rime/home#1735] [Q&A]
现在已经实现在 Obsidian 按下 ESC 进入 vim 的 normal mode 能自动切换为英文，我想问一下能不能实现这样一种需求：回到 insert mode 自动切换回中文？感谢！

```
md.obsidian:
     vim_mode: true
```

https://github.com/user-attachments/assets/98e1133d-6fcd-4381-9c9b-b61dff124809

**Comment by wangmuxi-luchi** [comment]
我是win10系统直接用obsidian的插件搞定了，https://github.com/ALONELUR/vim-im-select-obsidian，这个插件之前容易失灵，前段时间更新0.2.1过后好用多了
